components/raftstore-v2/src/operation/query/mod.rs

```diff
@@ -41,7 +41,7 @@ use crate::{
 };
 
 mod lease;
-mod local;
+pub(crate) mod local;
```

Comment:
Should re-export internal type instead of making the whole module public.

Reply:
Godd suggestion.